docs/libcurl/curl_url_get.3

```diff
@@ -76,8 +76,8 @@ Scheme cannot be URL decoded on get.
 .IP CURLUPART_PASSWORD
 .IP CURLUPART_OPTIONS
 .IP CURLUPART_HOST
-If the host part is an IPv6 numeric address, the zoneid will not be part of
-the extracted host but is provided separately in \fICURLUPART_ZONEID\fP.
+The host name. If it is an IPv6 numeric address, the zoneid will not be part of
+it but is provided separately in \fICURLUPART_ZONEID\fP.
```

Comment:
Perhaps it should also mention that the host name starts and ends with brackets `[]` if it contains an IPv6  numerical address. It is probably not obvious for everyone.